assembl/templates/views/adminTimelineEvent.tmpl

```diff
@@ -0,0 +1,42 @@
+<div class="subpanel-body bx">
+  <div>
+    <div class="admin-msg-elm">
+      <div class="control-group">
+        <label for="start_date" class="control-label">{{ gettext('Start date') }}</label>
+        <div class="controls">
+          <input class="js_start_date input-medium" id="start_date" name="start_date" type="datetime-local" value="<%= event.getStartNoTZ() %>"></input>
```

Comment:
No timezone: I do some postprocessing to add/remove the "Z", you may have missed it.
Partial support: well, it falls back to textfield behaviour, which is not that bad, and then it'll validate on fronted. I agree this is less than satisfactory, it was coded fast. Using moment.js would be a nice improvement.